app/backend/src/couchers/servicers/references.py

```diff
@@ -109,7 +109,7 @@ def ListReferences(self, request, context, session):
 
         to_users = aliased(User)
         from_users = aliased(User)
-        statement = select(Reference)
+        statement = select(Reference).where(Reference.is_deleted == False)
```

Comment:
Please also add this check in `user_model_to_pb` in `api.py`, line 796.

It might be good to add a `@hybrid_property` for `is_visible` in the Reference model and use that, but I don't think that's so important